WreckEngine/Game.h

```diff
@@ -14,16 +14,18 @@
 class Game
 {
 public:
-	Game() = default;
-	Game(GLprogram prog);
+    Game() = default;
+    Game(GLprogram prog);
+    virtual ~Game() = default;
 
-	void addState(shared<State> s);
+    void addState(shared<State> s);
 
-	virtual void update(double dt);
-	virtual void draw();
+    virtual void update(double dt);
+    virtual void draw();
 protected:
-	GLprogram shader;
-	std::vector<shared<State>> states;
-	State* currState;
+    GLprogram shader;
+    std::vector<shared<State>> states;
+    State* currState { nullptr };
+    bool drawDebug { true };
```

Comment:
Elsewhere uses the initializer sequence style, so that's why I used it. Unless you want to use equals style for basic types, then I'll go ahead and change it.